src/old/lib/utilities/API/api.ts

```diff
@@ -192,6 +192,44 @@ export const login = async (
   return instance.post('/auth/login', { email, password });
 };
 
+export const changePasswordRequest = async (
+  email: string,
+  password: string,
+): Promise<void> => {
+  // body {message: string}
+  await instance.post('/user/change-password', { email, password });
+};
+
+export const resetPasswordRequest = async (email: string): Promise<void> => {
+  await instance.post('/user/reset-password', { email });
+};
+
+export const newPasswordRequest = async (
+  token: string,
+  newPassword: string,
+  matchPassword: string,
+): Promise<boolean> => {
+  try {
+    const response = await instance.post('/user/update-password', {
+      token,
+      newPassword,
+      matchPassword,
+    });
+    return response.status === 200;
```

Comment:
what if response is not 200, but within 200-299 range? 

Better to check like this:

```suggestion
    return response.ok;
```